x-pack/plugins/enterprise_search/public/applications/applications/components/playground/playground.tsx

```diff
@@ -35,7 +46,7 @@ export const Playground: React.FC = () => {
         customPageSections
         bottomBorder="extended"
       >
-        <searchPlayground.Playground />
+        <searchPlayground.Playground pageMode={pageMode as unknown as PlaygroundPageMode} />
```

Comment:
If we updated `PlaygroundProps` to use `PlaygroundPageMode` we wouldn't need this type casting right?